x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/upgrade_prebuilt_rules_table/upgrade_prebuilt_rules_table_context.tsx

```diff
@@ -135,7 +137,10 @@ export const UpgradePrebuiltRulesTableContextProvider = ({
     isLoading,
     isRefetching,
   } = usePrebuiltRulesUpgradeReview({
-    refetchInterval: false, // Disable automatic refetching since request is expensive
+    // Refetch review prebuilt rules upgrade data in case of
```

Comment:
nit: this makes it sound as if we are refetching when any of those two events in the bullet points happen, which might be confusing for future devs. We could reword it like:

`// Refetch review prebuilt rules upgrade data to detect possible cases of:`